graphql_api/schema/custom/automation_task_base.py

```diff
@@ -45,6 +33,7 @@ def get_node(cls, info, _id):
     @staticmethod
     def from_json(jsondata):
         #Field type transforms...
+        started = dt.now()
```

Comment:
@benjamineac I don't think from_json should be setting started  ??

Reply:
I didn't touch this function.. this looks weird to me

Reply:
weird indieed - scuttle it!